pkg/sql/plan2/build_ddl.go

```diff
@@ -35,7 +35,7 @@ func buildCreateTable(stmt *tree.CreateTable, ctx CompilerContext) (*Plan, error
 
 	// get database name
 	if len(stmt.Table.SchemaName) == 0 {
-		createTable.Database = ctx.DefaultDatabase()
+		createTable.Database = ""
```

Comment:
maybe you can change to :

createTable.database = string(stmt.Table.SchemaName)

if we don't want to get default database.

Reply:
It will change back when DefaultDatabase() return right. Now, DefaultDatabase() will return "mo_catalog"

Reply:
default database should be current database, that is, the one you connected to (use database).

Reply:
@iamlinjunhong Are you saying DafaultDatabase() has a bug or NYI, and we will revert to ctx.DefaultDatabase() once we fix DefaultDatabase?    At least deserve a HUGE XXX comment.